index.bs

```diff
@@ -1194,9 +1197,15 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
         Please note that this is a greater relaxation of "same-origin" restrictions than what
         [=document.domain=]'s setter provides.
 
-        These restrictions on origin values apply to [=WebAuthn Clients=]. If other specifications mimic the [=Web Authentication API=] to make WebAuthn [=credentials=] usable on other platforms (e.g. native mobile applications), they might have different rules for binding a caller to a [=Relying Party Identifier=]. Such rules are outside the scope of this specification.
+        These restrictions on origin values apply to [=WebAuthn Clients=].
     </div>
 
+    Other specifications mimicking the [=web authentication api|WebAuthn API=] to enable WebAuthn [=public key credentials=] on non-Web platforms (e.g. native mobile applications), MAY define different rules for binding a caller to a [=Relying Party Identifier=].
+
+    Such specifications MAY also define their own [=RP ID=] syntax, e.g., arbitrary URLs [[!URL]], or even arbitrary strings. [=Public key credentials=] created with such non-[=valid domain string=] [=RP ID=]s MUST be [=non-discoverable credentials=].
+
+    Such rules are outside the scope of this specification.
```

Comment:
If these rules are outside the scope of this specification then why are we putting rules in the specification?

Reply:
yeah, i'll clean that up.